README.md

```diff
@@ -13,25 +13,18 @@ Technologies:
 - Express
 - React
 - Foundations CSS
-
-User Stories:
-Story 1:
-As a dog owner
-I want to see a list of dog parks
-So I can see which ones I could go to
-
-Story 2:
-As a dog owner
-I want to see reviews of those parks
-So I can choose a good dog park
-
-Story 3:
-As a dog owner
-I want to add new parks to the app
-So I can share the parks with other users
-
+- PostgreSQL
 
 Installation: 
+At first you will want to clone from github so you can get the app up and running on your end and to do so you would do these steps:
+- First browse to where you want to put this project from your terminal
+- After completing the first step you should clone the project from github with this command: `git clone https://github.com/kstrikis/Dog-Parks.git`
+- Make sure you are in your server folder before you run any of these commands below and you would get into your server using this command: `cd server`
+- create the database with the original assigned name that is on the project and you would do so using this command: `createdb dog-parks_developement`
+- You would yarn install with this command before you put your migrations up to date : `yarn install` 
+- You would run these commands to make sure that your migrations are up to date with the database : `yarn run migrate:latest && yarn run migrate:rollback && yarn run migrate:latest`
+- You run the following command to seed the database with initial data : `yarn db:seed`
+- You run the following commands to make sure that you connect your code to the server with : `yarn run dev`
```

Comment:
Love the attention to detail, but I would:
- where possible, remove the use of "you" e.g. "Run the `yarn db:seed` command`
- you can assume that the individual taking this code is familiar with npm, yarn, terminal commands and git. As such, a lot of these explanations can be cut down or removed e.g. remove "First browse to where you want to put this project from your terminal"
- grammatically things are a bit odd with "you would". Remove you, and prefer just "run yarn db:seed"
- trim down `yarn run migrate:latest && yarn run migrate:rollback && yarn run migrate:latest` to just `yarn migrate:latest`. Your dev work should make it so that others dont need to worry about whether or not a migration can be rolled back.
- make "Installation" a markdown header with `##`